src/main/java/com/hazelcast/aws/utility/EC2RequestSigner.java

```diff
@@ -192,7 +191,7 @@ private String sha256Hashhex(String in) {
     }
 
     public String createFormattedCredential() {
-        return config.getAccessKey() + '/' + timestamp.substring(0, LAST_INDEX) + '/' + config.getRegion() + '/'
+        return credentials.getAccessKey() + '/' + timestamp.substring(0, LAST_INDEX) + '/' + region + '/'
```

Comment:
`String.format` all of it?

Reply:
I'll address it in a separate PR while working on refactoring EC2RequestSigner.